emls/zktestemail_twitter.eml

```diff
@@ -1,404 +0,0 @@
-Delivered-To: zkemailverify@gmail.com

```

Comment:
Wait where did this file go, it's nice to have around

Reply:
This is test email for twitter - moved it to twitter repo